src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/Fixtures/DataSetFileMetaGeneratorExtensions.cs

```diff
@@ -15,6 +15,7 @@ public static Generator<DataSetFileMeta> WithDefaults(this Generator<DataSetFile
 
     public static InstanceSetters<DataSetFileMeta> SetDefaults(this InstanceSetters<DataSetFileMeta> setters)
         => setters
+            .SetNumDataFileRows(9393)
```

Comment:
I think you'd normally be able to just ask for a unique int here, like:

```
.SetDefault(d => d.NumDataFileRows)
```

That way you get something unique for each instance generated.  If you then wanted to test against a specific value, you could include a `WithNumDataFileRows()` to be able to provide it, which you have indeed included in this PR.